src/signHash.c

```diff
@@ -89,6 +98,63 @@ UX_FLOW(ux_signhash_flow,
   &ux_signhash_flow_4_step
 );
 
+void ui_display_sign_hash_flow(void) {
+	// Generate a string for the index.
+	snprintf(ctx->indexStr, sizeof(ctx->indexStr), "with Key #%d?", ctx->keyIndex);
+
+	ux_flow_init(0, ux_signhash_flow, NULL);
+}
+
+#else // HAVE_BAGL
+
+static nbgl_layoutTagValue_t pair;
+static nbgl_layoutTagValueList_t pairList = {0};
+static nbgl_pageInfoLongPress_t infoLongPress;
+
+static void transaction_rejected(void) {
+	do_reject();
+}
+
+static void reject_confirmation(void) {
+	nbgl_useCaseConfirm("Reject transaction?", NULL, "Yes, Reject", "Go back to transaction", transaction_rejected);
+}
```

Comment:
What is the purpose of wrapping `do_reject`? Is it related to the expected function signature?

Reply:
Looking at the signature of `do_reject` it appears to be the same as `transaction_rejected`: 

```
static void do_reject(void) {
```

Reply:
The signature is the same, so no great reason.
I would say to be closer to what is done in case of approval. But I can remove this wrapping if you want.

Reply:
I think either way is fine. I will leave it up to your judgement - I was mostly curious.